string command
int32 GameState
float32 B_X
float32 B_Y
float32 Locate1_X
float32 Locate1_Y
float32 Locate2_X
float32 Locate2_Y
float32 Locate3_X
float32 Locate3_Y
float32 Locate4_X
float32 Locate4_Y
float32 Locate5_X
float32 Locate5_Y
float32 Locate6_X
float32 Locate6_Y
---
int32 blueGoal 
int32 yellowGoal
int32 gameState
string stateHistory
string whosBall
